### PR TITLE
Itunes api

### PR DESCRIPTION
**Pull request by bhaveshpawar07**
### Ticket Link

---

### Related Links

---

### Description

---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### Live Link

---

- http://wednesday-react-template.s3-website.ap-south-1.amazonaws.com/<BRANCH_NAME>

### CODE REVIEW

**Review by naeemdadi1 — COMMENTED** [review]
resolve the merge conflict

File: app/containers/ITunes/selectors.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
+import { get } from 'lodash';
 import { createSelector } from 'reselect';
 import { initialState } from './reducer';
 
 /**
  * Direct selector to the iTunes state domain
  */
 
-const selectITunesDomain = (state) => state.iTunes || initialState;
+export const selectITunesDomain = (state) => state.iTunes || initialState;
```
Comment (naeemdadi1) [inline app/containers/ITunes/selectors.js:9]: why are we exporting this?

Comment (bhaveshpawar07) [reply app/containers/ITunes/selectors.js:9]: This has been exported inorder to  test it.

Comment (naeemdadi1) [reply app/containers/ITunes/selectors.js:9]: got it, thanks

**Pull request merged**